File: javascript/getQuotes.js
var http =  require('http');
var req = http.request('http://quotesondesign.com/wp-json/posts?filter[orderby]=rand&filter[posts_per_page]=1', function (res){
    var data = '';
        res.on('data', function( chunk ){
            data += chunk;
        });
        res.on('end', function(){
            console.log(data);
        });
    });
req.on('error', function (e) {
    console.log(e.message);
});
req.end();

File: java/getQuotes.go
package getQuotes;

import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
import java.net.HttpURLConnection;
import java.net.URL;

public class HttpURLCon {

	public static void main(String[] args) throws IOException {

		sendGET();
		System.out.println("GET DONE");
	}

	private static void sendGET() throws IOException {
		URL obj = new URL("http://quotesondesign.com/wp-json/posts?filter[orderby]=rand&filter[posts_per_page]=1;");
		HttpURLConnection con = (HttpURLConnection) obj.openConnection();
		con.setRequestMethod("GET");
		int responseCode = con.getResponseCode();
		System.out.println("GET Response Code :: " + responseCode);
		if (responseCode == HttpURLConnection.HTTP_OK) { // success
			BufferedReader in = new BufferedReader(new InputStreamReader(
					con.getInputStream()));
			String inputLine;
			StringBuffer response = new StringBuffer();

			while ((inputLine = in.readLine()) != null) {
				response.append(inputLine);
			}
			in.close();

			// print result
			System.out.println(response.toString());
		} else {
			System.out.println("GET request not worked");
		}

	}

}
